views/pages/SignInPage/index.jsx

```diff
@@ -30,7 +24,7 @@ const SignInPage = () => {
   }
 
   return (
-    <Box className="SignInPage_container" sx={{ boxShadow: 5 }}>
+    <>
```

Comment:
The different widths of the forms on this page makes it so that we use the fancy box div to display the width and content.

Reply:
we shouldn't have different width forms, the layout should be consistent. By putting the width in the layout we no longer need to specify a width here.

Reply:
We have a screen where the width is different

Reply:
this seems to be a mistake in the designs. Erica confirmed they should have consistent widths https://gnardog.slack.com/archives/C05DT214NHY/p1699556749283799?thread_ts=1699556549.223369&cid=C05DT214NHY